### PR TITLE
Feature/slade wilson indicators tsv

### PR DESCRIPTION
**Pull request by TheCookieLab**
Changes proposed in this pull request:
- **PriceVariationIndicator** renamed to **ClosePriceRatioIndicator** for consistency with new **ClosePriceDifferenceIndicator**
- added **ClosePriceDifferenceIndicator**
- added **TimeSegmentedVolumeIndicator**

- [X] added an entry with related ticket number(s) to the unreleased section of `CHANGES.md`

### CODE REVIEW

File: ta4j-core/src/main/java/org/ta4j/core/indicators/helpers/ClosePriceDifferenceIndicator.java
Comment (nimo23) [inline ta4j-core/src/main/java/org/ta4j/core/indicators/helpers/ClosePriceDifferenceIndicator.java:33]: Please always include a Javadoc description. In this case:

> The difference between the close price and the previous close price.

File: ta4j-core/src/main/java/org/ta4j/core/indicators/volume/TimeSegmentedVolumeIndicator.java
Comment (nimo23) [inline ta4j-core/src/main/java/org/ta4j/core/indicators/volume/TimeSegmentedVolumeIndicator.java:36]: Please add sections to the javadoc by adding a paragraph to each block for readability. In this case: Add a `<p>`-tag:

```
Time Segmented Volume (TSV) indicator.

<p>
This class calculates the Time Segmented Volume (TSV). TSV is a volume-based indicator that gauges the supply and demand for a security based on the price changes and volume of each period (or "segment") in the given time frame.

<p>
The calculation involves multiplying the volume of each period by the difference between the close price and the open price, and summing up these values over the given period.

@see ..
```

File: ta4j-core/src/main/java/org/ta4j/core/indicators/helpers/ClosePriceRatioIndicator.java
Comment (nimo23) [inline ta4j-core/src/main/java/org/ta4j/core/indicators/helpers/ClosePriceRatioIndicator.java]: Please add a period at the end of the sentence and a `<p>`-tag between both blocks:

```
/**
 * Price variation indicator.
 * 
 * <p>
 * The ratio between the current and previous close price.
 */
```

We should use this form for any indicator to be consistent.

Maybe @team172011 can add this form as a guideline in https://github.com/ta4j/ta4j-wiki/blob/master/How-to-contribute.md#coding-rules-and-conventions:

```
/**
 * NAME OF THE INDICATOR.
 * 
 * <p>
 * DESCRIPTION.
 * 
 * @see LINK (optonal)
*/
```

**Review by team172011 — APPROVED** [review]

**Comment by TheCookieLab** [comment]
A side effect of https://github.com/ta4j/ta4j/issues/965 is that approved PRs will need re-approval after updates to master as resolving merge conflicts (ex. CHANGELOG) resets approval status (despite the visual checkmark remaining)

**Comment by team172011** [comment]
> A side effect of #965 is that approved PRs will need re-approval after updates to master as resolving merge conflicts (ex. CHANGELOG) resets approval status (despite the visual checkmark remaining)

I change the branch protection rule so that this is not necessary anymore

**Pull request merged**
